src/Interfaces/ILanguage.ts

```diff
@@ -0,0 +1,9 @@
+export interface ILanguage {
+    sno: Number;
+    title: string;
+    langId: string;
+}
+
+export interface ILanguageListProps{
```

Comment:
Add a space between `ILanguageListProps` and `{`

Reply:
will do. FYI I said *Kindly ignore the convention problems. It's not detected by the eslint. I have to check that part. Help me with the above 2 problems*